### ISSUE TITLE
Add notification indicator when column is not visible

### ISSUE THREAD

**Issue by trwnh** [tootsuite/mastodon#8050]
Related: #710 
Context: https://github.com/tootsuite/mastodon/issues/710#issuecomment-377401303

The notification column can be unpinned in the desktop view, or it can be unfocused in the mobile view. In both of these cases, there is no indication when any new notifications are received. A small badge or color change should be added in these two cases.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by trwnh** [comment]
This is a thing in the new single-column mode, right? Just not in the multi-column mode?

**Comment by eobet** [comment]
Yes, I'm dearly missing it in multi-column. Just discovered I had several notifications waiting for me.
